common/persistence/persistenceMetricClients.go

```diff
@@ -436,6 +436,9 @@ func (p *workflowExecutionPersistenceClient) ListCurrentExecutions(
 	op := func() error {
 		var err error
 		resp, err = p.persistence.ListCurrentExecutions(ctx, request)
+		if err == nil && len(resp.Executions) == 0 {
+			p.metricClient.IncCounter(metrics.PersistenceListCurrentExecutionsScope, metrics.PersistenceEmptyResponseCounter)
```

Comment:
Instead of creating a new scope, you need to add `operation` tags to your metrics. That would make the implementation simpler and we can both aggregate and separate empty responses by operations

Reply:
We have operation tags binding with the existing scopes in defs.go, for this one, it has `PersistenceListCurrentExecutionsScope: {operation: "ListCurrentExecutions"},`

Reply:
Oh I see, these are existing scopes.